src/libsyntax/parse/diagnostics.rs

```diff
@@ -17,6 +17,8 @@ use syntax_pos::{Span, DUMMY_SP, MultiSpan, SpanSnippetError};
 use log::{debug, trace};
 use std::mem;
 
+const TURBOFISH: &'static str = "use the \"turbofish\" `::<...>` instead of `<...>` to specify \
```

Comment:
I would prefer if we kept the "turbofish" jargon out of any official document or diagnostic message. cc @rust-lang/compiler

Reply:
I tend to agree but I don't feel strongly myself.

Reply:
That's what I was doing originally, but are otherwise not giving the user anything to Google for if they want to understand further. Of course, I could add a link to the book instead.

Reply:
Instead of meme-ing around the turbofish we could've called it "generic arguments disambiguator `::`" or something similar that has some chance of hitting consistent search results without being entirely opaque.